src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
```

Comment:
Should these be applied to the base style above instead of only for the medium breakpoint?

Reply:
The greyscale Sign icon is only meant to show up on medium screen sizes and below, so it's going to be hidden on anything bigger. Functionally, it would end up being the same. I'll go ahead and update it if it's more clear that way.